compiler/compiler.go

```diff
@@ -15,6 +15,12 @@ func Compile(scope Scope, f *semantic.FunctionExpression, in semantic.MonoType)
 		return nil, errors.Newf(codes.Invalid, "function input must be an object @ %v", f.Location())
 	}
 
+	// If the function is vectorizable, `f.Vectorized` will be populated, and
+	// we should use the FunctionExpression it points to instead of `f`
+	if f.Vectorized != nil {
```

Comment:
What mechanism will be exposed so the caller knows they're using the vectorized version?